.cargo/config.toml

```diff
@@ -12,7 +12,74 @@ rustflags = [
 	# rustc additional warnings:
 	"--warn",
```

Comment:
I really wish this syntax would allow `"--warn clippy::await_holding_lock",` on a single line 😞

But since we are using `-W` everywhere below, I suggest changing the above as well.